src/Domain/Creators/TimeDependence/ScalingAndZRotation.hpp

```diff
@@ -164,9 +164,8 @@ class ScalingAndZRotation final : public TimeDependence<MeshDim> {
           std::string,
           std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>> override;
 
-  /// Returns the map for each block to be used in a composition of
-  /// `TimeDependence`s.
-  MapForComposition map_for_composition() const;
+  /// Returns the map for a single block.
+  GridToInertialMap grid_to_inertial_map() const;
```

Comment:
it's been a while, so I don't remember all the details, but can this function be removed completely now or is/will it (be) used? I see it's used in some tests, but that might have just been for the sake of testing that it would (hopefully) work in a composition. Maybe the function could be private since it's also used to reduce code duplication in some of the TimeDepedences?

Reply:
I like making it private and removing it from the tests.  Unless I implement this as a poor-mans Composition (see Kyle's suggestion above) in which case they need to stay public.